x11-themes/kvantum/Manifest

```diff
@@ -1,2 +1,6 @@
+AUX kvantum-disable-bundled-themes.patch 288 BLAKE2B 4b816f1f8ef96509a591d473b6c901d0bc48984936b6a35110e24311825c3f4336ddc1f9773bec34115ffd83a5f9af124ede2581f5b5de1fe3d4cf438a70e014 SHA512 0e59c139aba0037211836d51f35335476ab253a0be108e61f1eb742f5a5a5388b76d8b486a3924ed1c285bdcabd4057af5fa13b8a1a5b70e219bf1ae55573864
 DIST kvantum-0.14.1.tar.gz 2461477 BLAKE2B 6c7fd45801eff1183ee46a157b5c31591da31c9b5fa9586b6095074a272fa705ac826733ce45a679b857276fbd358fdb56dc72ffe6c4744557e25c9c6782ca4a SHA512 18faa895bb6affa1744d15aa5b64636c49f7780eddca62ecee18c133702a156bca055cda473af9ea72e1d6de17fd86c4d7212bd16c544030d96d8afa4f5aff69
 DIST kvantum-0.15.2.tar.gz 2652167 BLAKE2B 610bd705d40605cf7d14ffe9acc62bcd276159ffd6f12c315871d633be74d986efc6c45ac938b26851af78565140e2a0dc0424174c543a0622dbf9c951225668 SHA512 98b83a14c3c86636acb6f2440d64dc47496536b5487dc08aed8e8370d656e8838d00866bfb54a4b5752135bdc3d3da0eaab209f5ca3e33c237aada59640a78f7
+EBUILD kvantum-0.14.1.ebuild 841 BLAKE2B dc3c3abc812bfd579bf7171d747c1714d7ec32413ed4a38267287dbd9b0144c193cca3bcb0799fff19afc25a825dbf2467745ddb5e61b87add8dd8f1226a9aba SHA512 091313fec78a21c118a6c8de0244de9aa3e265668045e86b8463aa09ee843fd0e805e50e6fdeafe18cdb212fdfea9bd84e83c77f454e3e129a585d9e431fc5e0
```

Comment:
You should configure your repository to use thin manifests. These *.ebuild files caused CI breakage.

https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests

Reply:
Fixed, thanks